core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -778,6 +788,7 @@ public InodeDirectory getRoot() {
               .setLastModificationTimeMs(context.getOperationTimeMs())
               .setLastAccessTimeMs(context.getOperationTimeMs());
           if (context.getXAttr() != null) {
+            // TODO(czhu): determine if xAttr update strategy is required for this
```

Comment:
Currently we will just rely on the prior behaviour, which is to truncate any existing Inode `xAttr` data.